common/types/option.go

```diff
@@ -113,7 +113,7 @@ func (o *BoolOptions) SetIfUnset(key string, value bool) {
 }
 
 func (o *BoolOptions) InheritDefault(parent *BoolOptions, key string) {
-	o.SetIfUnset(key, parent.IsEnabled(key))
+	o.Set(key, parent.IsEnabled(key))
```

Comment:
We can get rid of `InheritDefault` alltogether and just call `Set()` in `SetDefaultOpts()`